appserver/connectors/connectors-runtime/src/main/java/com/sun/enterprise/resource/pool/ConnectionPool.java

```diff
@@ -67,38 +68,136 @@ public class ConnectionPool implements ResourcePool, ConnectionLeakListener, Res
     private static final Logger LOG = LogDomains.getLogger(ConnectionPool.class, LogDomains.RSR_LOGGER);
 
     // pool life-cycle config properties
+    /**
+     * Represents the "max-wait-time-in-millis" configuration value.<br>
```

Comment:
I guess it's about different config val.

Reply:
Fixed in commit [9208483](https://github.com/eclipse-ee4j/glassfish/pull/24879/commits/92084835a8cd6365a37910d96ee4d435e473e3ab)